features/inventory/checkout_inventory/validating_product_quantity_on_checkout_complete.feature

```diff
@@ -16,23 +16,23 @@ Feature: Being unable to buy products that are out of stock
         And the store allows paying offline
         And I am a logged in customer
 
-    @ui
+    @ui @api
     Scenario: Placing an order with products that have sufficient quantity
         Given I have added 3 products "Iron Maiden T-Shirt" to the cart
-        And I have proceeded selecting "Offline" payment method
+        And I proceed through checkout process
```

Comment:
```suggestion
        When I proceed through checkout process
```
but why did you change that step?

Reply:
I had a problem with addresses, in behats they were empty, and I get SM error, with this step, the scenario is still valid and checkout works as I expect.